Microsoft.Toolkit.Uwp.UI.Controls/Carousel/Carousel.cs

```diff
@@ -184,6 +184,11 @@ private static void OnCarouselPropertyChanged(DependencyObject d, DependencyProp
 
             if (e.Property == SelectedIndexProperty)
             {
+                if (carouselControl.Items.Count < (int)e.NewValue + 1)
```

Comment:
This seems weird, can you rephrase it? (int)e.NewValue >= carouselControl.Items.Count